service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -573,6 +575,42 @@ void shouldNotifyLocalAuthorityOfIssuedStandardDirectionsOrder() throws IOExcept
             eq("12345"));
     }
 
+    @Nested
+    class NoticeOfPlacementOrderNotification {
+        private NotificationHandler placementNotificationHandler;
+
+        @BeforeEach
+        void setup() throws IOException {
+            given(inboxLookupService.getNotificationRecipientEmail(
+                callbackRequest().getCaseDetails(), LOCAL_AUTHORITY_CODE)).willReturn(LOCAL_AUTHORITY_EMAIL_ADDRESS);
+
+            placementNotificationHandler = new NotificationHandler(hmctsCourtLookupConfiguration,
+                cafcassLookupConfiguration, hmctsEmailContentProvider, cafcassEmailContentProvider,
+                cafcassEmailContentProviderSDOIssued, gatekeeperEmailContentProvider, c2UploadedEmailContentProvider,
+                orderEmailContentProvider, localAuthorityEmailContentProvider, notificationClient, idamApi,
+                inboxLookupService, caseManagementOrderEmailContentProvider, placementApplicationContentProvider,
+                representativeService, localAuthorityNameLookupConfiguration, objectMapper);
+        }
+
+        @Test
+        void shouldSendNotificationForPlacementOrderUploaded() throws IOException, NotificationClientException {
+            Map<String, Object> parameters = Map.of("respondentLastName", "Nelson",
+                "caseUrl", String.format("%s/case/%s/%s/%s", "http://fake-url", JURISDICTION, CASE_TYPE, 1L));
+
+            given(localAuthorityEmailContentProvider.buildNoticeOfPlacementOrderUploadedNotification(
+                callbackRequest().getCaseDetails())).willReturn(parameters);
```

Comment:
Maybe extract `callbackRequest()` to a variable seems to be used in a lot of places

Reply:
hmm I have only added one new test, this is existing stuff that I have just changed slightly. Don't want to do any more work on it